cfme/tests/networks/nuage/test_inventory.py

```diff
@@ -18,3 +18,39 @@ def test_tenant_details(setup_provider_modscope, provider, with_nuage_sandbox_mo
         'num_security_groups': 2,
         'num_network_ports': 4
     })
+
+
+def test_subnet_details(setup_provider_modscope, provider, with_nuage_sandbox_modscope):
```

Comment:
Please add a docstring to inform user what the difference between L3 and L2 subnet is so he won't be scrathing head why we go via Router here, but not in the next test:

```python
"""
Ensure L3 subnet details displays expected info.

L3 subnets are always connected to routers, hence we navigate to them as
  Tenant > Router > Subnet
"""
```